static-assets/components/cstudio-admin/mods/content-types.js

```diff
@@ -1359,7 +1362,8 @@
             if (!deleteEl) {
               deleteEl = document.createElement('i');
               YDom.addClass(deleteEl, 'deleteControl fa fa-times-circle');
-              listeningEl.appendChild(deleteEl);
+
+              !defaultField && listeningEl.appendChild(deleteEl);
```

Comment:
Would this not disable the elimination of the `file-name` control? (e.g. when one wants to use a auto-filename instead of a file name)

Reply:
@jvega190 👆 ?

Reply:
@rart Yeah, it was part of the ticket https://github.com/craftercms/craftercms/issues/3825, it may need to be discussed

Reply:
Adding filename switcher to autofilename on components (not pages)

Reply:
@rart default auto-filename control was already there for components